setup.py

```diff
@@ -37,6 +37,8 @@ def find_version(*file_paths):
         "Development Status :: 4 - Beta",
         'Environment :: Web Environment',
         'Framework :: Django',
+        'Framework :: Django :: 1.10',
+        'Framework :: Django :: 1.11',
```

Comment:
Welp, I tried to run the tests for this package and I ran into more problems. So starting with what is here: https://github.com/caktus/django-sorter/blob/develop/Makefile

I setup my ccsr package to run the sorter tests:

    python manage.py test --settings=sorter.test_settings sorter

There was an error right away about `add_to_bultins` not existing anymore (see https://docs.djangoproject.com/en/2.0/releases/1.9/#django-template-base-add-to-builtins-is-removed).

I've twiddled with these tests a little more (they still don't pass for me yet)...but I think they're close.

---

Long story short:
 * Add support for django 1.9 to this section too!
 * get the test cases to pass (I'm happy to take this on if you don't mind me pushing to your PR).

Reply:
k - I pushed the updates to get the test cases working (hope you don't mind about that - I figured it might be a little faster since I already got started playing with that part).